docs/static/monitoring/monitoring-ea-dashboards.asciidoc

```diff
@@ -71,8 +71,8 @@ details.
 * Under **Logs**, modify the log paths to match your {ls} environment.
 
 . Configure the integration to collect metrics.
-* Make sure that **Metrics (Technical Preview)** is turned on, and **Metrics (Stack Monitoring)** is turned off.
-* Under **Metrics (Technical Preview)**, make sure the {ls} URL setting
+* By default **Metrics (Elastic Agent)** is enabled, and **Metrics (Stack Monitoring)** is turned off.
```

Comment:
Just a suggestion but I'd prefer to keep it as an action item, instead of an observation:

```suggestion
* Make sure **Metrics (Elastic Agent)** is enabled (default), and **Metrics (Stack Monitoring)** is turned off.
```

Feel free to ignore the suggestion. otherwise LGTM